osu.Game/Overlays/Mods/ModSelectScreen.cs

```diff
@@ -50,6 +49,15 @@ public abstract class ModSelectScreen : ShearedOverlayContainer
             }
         }
 
+        /// <summary>
+        /// Hotkey that should be used to hide the mod select screen.
+        /// </summary>
+        /// <remarks>
+        /// This is handled locally here rather than via <see cref="FooterButtonMods"/>, because this overlay is being registered at the game level
+        /// and therefore takes away keyboard focus from the screen stack.
+        /// </remarks>
+        internal GlobalAction? Hotkey { get; set; }
```

Comment:
Is there a case where this isn't set?

Reply:
In multiplayer, `RoomSubScreen` to be exact. Would be weird to have F1 work as a toggle binding there I think?

Reply:
On testing, this can actually get weird with freemod select overlay, since on master, pressing F1 while on the freemod overlay would close it and open the regular one. While here, pressing F1 will instead just switch to the previous track:

https://user-images.githubusercontent.com/22781491/167262299-814d509d-ad5c-480d-ae10-3e975e091f0c.mp4

Not sure if hardcoding will help or make things worse, though.

Reply:
I actually think I'd prefer to have it work there. I've heard some people complain they can't find the local mods button, and the first thing I'd try as a user in that scenario is hitting F1.

It doesn't look to conflict, so I think i'll hardcode this?

Reply:
I guess it won't actually work though, funnily enough (only works in the outwards direction). Kinda weird.

Reply:
The footer buttons are handling open. The mod select itself is only locally handling close, because it is a focused overlay that takes keyboard focus away from the screen behind it.

Reply:
I'm going to hard code this anyway. Will track the showing part at https://github.com/ppy/osu/issues/18140.